styles.css

```diff
@@ -1,11 +1,12 @@
 @import "./library/component.css";
 
 
-:root{
+:root {
     --grey-secondary: #757575;
 
 }
-body{
+
+body {
     background: #dedede;
```

Comment:
use var

Reply:
Specific to documentation site.
Only being used once, hence didn't use variables.